server/pyproject.toml

```diff
@@ -23,10 +23,10 @@ sentencepiece = "^0.2.0"
 datasets = { version = "^2.15.0", optional = true }
 texttable = { version = "^1.7.0", optional = true }
 # transformers pinned due to breakage from using `Cache` class for KV cache
-transformers = "4.40.2"
+transformers = "4.50.0"
```

Comment:
As per the comment, upgrading the `transformers` causes a breakage with how the TGIS code manages the KV Cache.... fixing that would likely require significant code changes.

Reply:
Reverted back the transformers to "4.40.2"